mmv1/products/filestore/Instance.yaml

```diff
@@ -98,6 +104,19 @@ properties:
       Possible values include: STANDARD, PREMIUM, BASIC_HDD, BASIC_SSD, HIGH_SCALE_SSD, ZONAL, REGIONAL and ENTERPRISE
     required: true
     immutable: true
+  - !ruby/object:Api::Type::Enum
+    name: 'protocol'
+    description: |
+      Either NFSv3, for using NFS version 3 as file sharing protocol,
+      or NFSv4.1, for using NFS version 4.1 as file sharing protocol.
+      NFSv4.1 can be used with HIGH_SCALE_SSD, ZONAL, REGIONAL and ENTERPRISE.
+      The default is NFSv3.
+    default_value: :NFS_V3
```

Comment:
This isn't returned by resources where it wasn't set explicitly, and null (as in nothing, not an explicit null) is returned instead. Can you add `default_if_empty: true` to the field? That'll rewrite null into this default value.

Reply:
From what I understand the API returns an empty value when the default value is set automatically and this generates a permadiff. To fix it I inserted default_if_empty.erb as custom_flatten. I hope I understood the hint.

Reply:
Yep! That's the correct thing to do, I forgot we hadn't promoted that from a custom_flatten to a dedicated override yet.

Reply:
n.p, ctrl+shift+f and documentation helped me to figure it out 😂